tests/admin_filters/tests.py

```diff
@@ -140,12 +140,24 @@ class CustomUserAdmin(UserAdmin):
 
 
 class BookAdmin(ModelAdmin):
-    list_filter = ('year', 'author', 'contributors', 'is_best_seller', 'date_registered', 'no')
+    list_filter = ('year', 'author', 'contributors', 'is_best_seller', 'date_registered', 'no', 'availability')
```

Comment:
This change is not required, please revert it.

Reply:
Reverted this unrelated change.